kauri/kauri_test.go

```diff
@@ -0,0 +1,64 @@
+package kauri
+
+import (
+	"testing"
+
+	"github.com/relab/hotstuff"
+	"github.com/relab/hotstuff/crypto"
+	"github.com/relab/hotstuff/crypto/ecdsa"
+)
+
+func TestSubTree(t *testing.T) {
+	subTreeTestData := []struct {
+		test_a   []hotstuff.ID
```

Comment:
This stutters when used `test.test_a`. Change to just `a` and `b`...

I prefer the shorter `want` instead of `expected`.